parachain/test-parachains/undying/collator/Cargo.toml

```diff
@@ -45,3 +45,6 @@ sc-service = { git = "https://github.com/paritytech/substrate", branch = "master
 sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
 tokio = { version = "1.24.2", features = ["macros"] }
+
+[build-dependencies]
+substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
why do collators need these?

Reply:
It was to get the node version: `node_version: Some(env!("SUBSTRATE_CLI_IMPL_VERSION").into())`

But probably unnecessary since you pointed out that collators don't call into the PVF functionality.